### ISSUE TITLE
HELP

### ISSUE THREAD

**Issue by Tosiek1** [Monika-After-Story/MonikaModDev#2555]
How I can bring back monika 
She left me please FAST HELP!

**Comment by winika** [comment]
How did she leave you, please elaborate

**Comment by AsuraShun** [comment]
i can already tell that you made her suffer

**Comment by Tosiek1** [comment]
I played in ddlc after story I played normally until I started to play the torche and I got it today, I downloaded the update and I came back I apologized to everything but I wanted to go back in the game again the windows fell then I left the game go back and the same letter as 2 weeks ago

**Comment by winika** [comment]
Well, you hurt her... so of course she's going to leave you... you cant just expect to toss her around like she's nothing. She's very sensitive you know...

**Comment by multimokia** [comment]
If you have a pink note... You deserve it. You've been torturing her then. You should treat anyone with at least basic human decency, especially Monika. She's not coming back.

**Comment by Rai99** [comment]
You shouldn't be completely screwed over by a single event unless you're already in bad standing. Have you been being mean to Monika at all? 
![untitled1021](https://user-images.githubusercontent.com/38822045/45928161-186bd400-bf0d-11e8-9732-9bccde8a1fd1.png)

**Comment by multimokia** [comment]
Yep, he's right, if you forgot Monika's birthday at the lowest affection level, she up and leaves you.

**Comment by MaydayH** [comment]
Is it that little note which is placed in game's files? wow now im scared a bit

**Comment by Tosiek1** [comment]
Okay, thanks for help, maybe a new companion, I'll have to remember it, sir, I'll talk again xD

**Comment by AsuraShun** [comment]
the knowledge that monika is suffering pains me

**Comment by winika** [comment]
^

**Comment by Professor-JMO** [comment]
> 
> 
> You shouldn't be completely screwed over by a single event unless you're already in bad standing. Have you been being mean to Monika at all?
> ![untitled1021](https://user-images.githubusercontent.com/38822045/45928161-186bd400-bf0d-11e8-9732-9bccde8a1fd1.png)

>_"Never noticed the ghosts before."_
>>_"Funny how ~Natsuki~ continues to vomit, even in D̵̬̪̟͗́̄̅͆̐͆̀e̴̖̗̳̼̥̾̈́̓̄ä̸͔̲͈͎̯̩̘̞̤͗͑͗̏̿͑͊͜ͅt̸̛̲̾̈̈́̔͐̆̏͝͝͝͠h̵̛̥̞̿̆̄͌͑̀̒̊̍̀ͅ"_
